tests/Microsoft.Identity.Test.Unit.net45/CoreTests/InstanceTests/AadAuthorityTests.cs

```diff
@@ -194,6 +194,8 @@ public void FailedValidationTest()
                 Authority instance = Authority.CreateAuthority("https://login.microsoft0nline.com/mytenant.com", true);
                 Assert.IsNotNull(instance);
                 Assert.AreEqual(instance.AuthorityInfo.AuthorityType, AuthorityType.Aad);
+
+                TestCommon.CreateServiceBundleWithCustomHttpManager(harness.HttpManager, authority: instance.AuthorityInfo.CanonicalAuthority, validateAuthority: true);
```

Comment:
is this enough? I think you have to take the service bundle that you create here and use it to build that resolver object at line 199

Reply:
This is only working because you changed the default ValidateAuthority in TestCommon from `false` to `true`. I think this new default is ok, but I don't think any other test changes are needed.